src/command.coffee

```diff
@@ -25,7 +25,7 @@ hidden = (file) -> /^\.|~$/.test file
 
 # The help banner that is printed in conjunction with `-h`/`--help`.
 BANNER = '''
-  Usage: coffee [options] [--] path/to/script.coffee [args]
```

Comment:
I don't like removing `[--]`, the new version does not say anything about how `--` is parsed as an argument. `gawk --help` uses `[--]` in a very similar way.